src/libs/fvcams/Makefile

```diff
@@ -41,7 +41,9 @@ OBJS_libfvcams = camera.o          \
                  cam_exceptions.o  \
                  factory.o         \
                  control/factory.o
-
+ifeq ($(HAVE_OPENCV),0)
```

Comment:
This is needed as for some reasone including fvconf.mk in Makefiles inverts the HAVE_OPENCV logic

Reply:
I don't think that's correct, I think your problem is this:
https://github.com/fawkesrobotics/fawkes/blob/95144fe94aaf41da255106d60cf7ed291136c167/etc/buildsys/fvconf.mk#L120-L123